setup.py

```diff
@@ -86,13 +86,23 @@
         "janus",
         "PySocks",
     ],
-    python_requires="!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5, <4",
+    python_requires=">=3.6, <4",
     packages=find_packages(
         exclude=[
```

Comment:
>exclude [](http://example.com/codeflow?start=8&length=7)

Is there a way to do this with `includes` instead of `excludes`? I may be wrong, but I assume we have to be careful to update setup.py every time we add a root-level folder, otherwise it will show up in the  package.

Reply:
It seems there is, but I wanted to err on the side of caution, given that we've only tested the use of "excludes" before. I also don't love this, so I can give "includes" a try, I'll just need to verify the behavior.

Reply:
Update: decided to go with a modified `src` pattern where the `src` folder is just named `azure-iot-device`. This allows me to more easily distribute only the directories we want.

Additionally, it's more similar to the existing structure, so less mess when merging with WIPs